core/src/main/java/bisq/core/api/model/CanceledTradeInfo.java

```diff
@@ -0,0 +1,55 @@
+package bisq.core.api.model;
```

Comment:
I just recognized that lots of files are missing the Bisq comment on top.

Reply:
I just recognized it wasn't configured in Codacy. Have changed that now, so it is checked automatically.

Reply:
License banners added in commit d902e77710bd7ff2236688fd105fdfc85a065263.